test/utils/hex.hpp

```diff
@@ -37,7 +37,7 @@ inline namespace literals
 /// Operator for "" literals, e.g. "0a0b0c0d"_bytes.
 inline fizzy::bytes operator""_bytes(const char* literal, size_t /*length*/)
 {
-    return fizzy::from_hex(literal);
+    return fizzy::test::from_hex(literal);
```

Comment:
looks like it could be just
```suggestion
    return from_hex(literal);
```

Reply:
Actually no, because this is in a difference namespace (`literals`).

Reply:
But `literals` is in `fizzy::test`.